locale/en/LC_MESSAGES/django.po

```diff
@@ -895,118 +891,106 @@ msgstr ""
 msgstr "to supplement a grant application"
 
-#: main/templates/main/index.html:114
+#: main/templates/main/index.html:111
 msgid "die al goedgekeurd is door een andere ethische toetsingscomissie."
 msgstr "that has been approved by another ethics committee."
 
-#: main/templates/main/index.html:120
+#: main/templates/main/index.html:117
 msgid "Een aanvraag reviseren"
 msgstr "Revise a proposal"
 
-#: main/templates/main/index.html:125
+#: main/templates/main/index.html:122
 msgid "als een revisie, gebaseerd op opmerkingen van de FETC-GW;"
 msgstr "as a revision, based on comments of the FEtC-H;"
 
-#: main/templates/main/index.html:128
+#: main/templates/main/index.html:125
 msgid "als amendement, wanneer de aanvraag al goedgekeurd is door de FETC-GW."
 msgstr ""
 "as an amendment, after your proposal has already been approved by the FEtC-H."
 
-#: main/templates/main/index.html:136
+#: main/templates/main/index.html:133
 msgid "Bekijk"
 msgstr "View"
 
-#: main/templates/main/index.html:140
+#: main/templates/main/index.html:137
 msgid "mijn conceptaanvragen;"
 msgstr "my draft applications;"
 
-#: main/templates/main/index.html:143
+#: main/templates/main/index.html:140
 msgid "mijn oefenaanvragen;"
 msgstr "my practice applications;"
 
-#: main/templates/main/index.html:146
+#: main/templates/main/index.html:143
 msgid "mijn ingediende aanvragen;"
 msgstr "my submitted applications;"
 
-#: main/templates/main/index.html:149
+#: main/templates/main/index.html:146
 msgid "mijn afgehandelde aanvragen;"
 msgstr "my processed applications;"
 
-#: main/templates/main/index.html:152
+#: main/templates/main/index.html:149
 msgid "mijn aanvragen als eindverantwoordelijke;"
 msgstr "my supervised applications;"
 
-#: main/templates/main/index.html:155
+#: main/templates/main/index.html:152
 msgid "al mijn aanvragen."
 msgstr "all my applications"
 
-#: main/templates/main/index.html:159
+#: main/templates/main/index.html:156
 msgid "FETC-GW archief"
 msgstr "FEtC-H archive"
 
-#: main/templates/main/index.html:163
+#: main/templates/main/index.html:160
 msgid "Alle goedgekeurde aanvragen bekijken van de Algemene Kamer"
 msgstr "View all processed and approved applications of the General Chamber"
 
-#: main/templates/main/index.html:166
+#: main/templates/main/index.html:163
 msgid "Alle goedgekeurde aanvragen bekijken van de Linguïstiek Kamer"
 msgstr ""
 "View all processed and approved applications of the Linguistics Chamber"
 
-#: main/templates/main/index.html:185
-msgid "Reglement Linguïstiek Kamer (LK)"
-msgstr "Regulations Linguistics Chamber (LC)"
+#: main/templates/main/index.html:177
+#: proposals/templates/proposals/proposal_start_pre_approved.html:14
+msgid "https://fetc-gw.wp.hum.uu.nl/reglement-fetc-gw/"
+msgstr "https://fetc-gw.wp.hum.uu.nl/en/regulations-fetc-h/"
 
-#: main/templates/main/index.html:192
+#: main/templates/main/index.html:178
+msgid "Reglement van de FETC-GW"
+msgstr "Regulations of the FEtC-H"
```

Comment:
The WP site calls these 'Regelement FETC-GW' and 'Regulations FEtC-H'. It might be nice to be consistant with those